src/features/classification/__tests__/Classification.test.js

```diff
@@ -57,6 +57,38 @@ describe('features/classification/Classification', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
+    test('should not render classification last modified information when modified props are not provided', () => {
+        const wrapper = getWrapper({
+            name: 'Confidential',
+            definition: 'fubar',
+            messageStyle: 'inline',
+        });
+        expect(wrapper.find('.bdl-Classification-modifiedBy').length).toBe(0);
```

Comment:
We should try to use `data-testid` attributes or component names to select elements, when possible. The `wrapper.exists` helper is also nice here to avoid the length check.